components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -828,6 +828,10 @@ macro_rules! repeated_vec_impl {
         $(impl<T: RepeatableListAnimatable> Animatable for $ty {
             fn add_weighted(&self, other: &Self, self_portion: f64, other_portion: f64)
                 -> Result<Self, ()> {
+                // If the length of either list is zero, the least common multiple is undefined.
+                if cmp::min(self.len(), other.len()) < 1 {
```

Comment:
Oh I thought this had already landed, that was just a FYI, don't worry about it.